server/routerlicious/packages/protocol-base/src/index.ts

```diff
@@ -5,6 +5,17 @@
 
 export { buildGitTreeHierarchy, getGitMode, getGitType } from "./gitHelper";
 export { IProtocolHandler, IScribeProtocolState, ProtocolOpHandler } from "./protocol";
+export {
+	TypedEventEmitter,
+	TypedEventTransform,
+	IEvent,
+	IEventProvider,
+	IEventTransformer,
+	TransformedEvent,
+	EventEmitterEventType,
+	IEventThisPlaceHolder,
+	ReplaceIEventThisPlaceHolder,
+} from "./typedEventEmitter";
```

Comment:
Since the currently exported `QuorumClients` is built on top of `TypedEventEmitter`, moving the latter here means that it and all its building blocks need to be exported.